src/cargo/core/features.rs

```diff
@@ -1107,6 +1109,7 @@ impl CliUnstable {
             "credential-process" => stabilized_warn(k, "1.74", STABILIZED_CREDENTIAL_PROCESS),
             "lints" => stabilized_warn(k, "1.74", STABILIZED_LINTS),
             "registry-auth" => stabilized_warn(k, "1.74", STABILIZED_REGISTRY_AUTH),
+            "check-cfg" => stabilized_warn(k, "1.80", STABILIZED_CHECK_CFG),
```

Comment:
We're currently on 1.79 so for this value, we need to wait a couple weeks (exact time depends on when beta is branched)

Reply:
According to [forge.rust-lang.org](https://forge.rust-lang.org/#current-release-versions) nightly will branch Friday next week (April 26 2024), which I think is a bit short, in particular since I suspect that the Cargo update into rust-lang/rust may take some time to be ready (not everything is using `-Zcheck-cfg`, so there might need to modify/update some parts).

We will need also need to update some links emitted by `rustc`, which will also add some delay and I'm not sure T-compiler will like having to backport if it misses the window.

Reply:
Not saying we need to squeeze this in.  If nothing else, we need to make sure we don't merge this too soon and want to make this specific requirement more explicit than this being in Draft

Reply:
We are now on 1.80, so this concern is no more.